consensus/consensus.go

```diff
@@ -25,38 +25,30 @@ const (
 // Consensus is the main struct with all states and data related to consensus process.
 type Consensus struct {
 	Decider quorum.Decider
-
 	// FBFTLog stores the pbft messages and blocks during FBFT process
 	FBFTLog *FBFTLog
 	// phase: different phase of FBFT protocol: pre-prepare, prepare, commit, finish etc
 	phase FBFTPhase
 	// current indicates what state a node is in
 	current State
-
 	// epoch: current epoch number
 	epoch uint64
-
 	// blockNum: the next blockNumber that FBFT is going to agree on,
 	// should be equal to the blockNumber of next block
 	blockNum uint64
 	// channel to receive consensus message
 	MsgChan chan []byte
-
```

Comment:
why remove the empty line here?

Reply:
because code density on page makes it easier to code, more information on screen at a time